wgengine/router/router_userspace_bsd.go

```diff
@@ -155,7 +124,13 @@ func (r *userspaceBSDRouter) Set(cfg *Config) (reterr error) {
 		}
 	}
 	for _, addr := range r.addrsToAdd(cfg.LocalAddrs) {
-		arg := []string{"ifconfig", r.tunname, inet(addr), addr.String(), addr.IP.String()}
+		var arg []string
+		if runtime.GOOS == "freebsd" && addr.IP.Is6() && addr.Bits == 128 {
```

Comment:
Add a comment explaining why you're changing the prefix length?

Reply:
Could drop a link to the bug too for people to enjoy the history.

Reply:
Added.